configargparse.py

```diff
@@ -673,7 +676,7 @@ def get_command_line_key_for_unknown_config_file_setting(self, key):
 
         return command_line_key
 
-    def get_items_for_config_file_output(self, source_to_settings,
+def get_items_for_config_file_output(self, source_to_settings,
```

Comment:
looks like accidentally deleted indent

Reply:
also please make sure all unittests pass
python3 -m unittest tests.test_configargparse
https://travis-ci.org/github/bw2/ConfigArgParse/builds/759318560

Reply:
Done.

Reply:
looks like I will need some assist to better understand the test